File: source/examples/Schemas/ProjectAndTask.java
// ProjectTask.java
public class ProjectTask extends RealmObject {
    @Required
    public String name;
    public String assignee;
    public int progressMinutes;
    public boolean isComplete;
    public int priority;
}

// Project.java
public class Project extends RealmObject {
    @Required
    public String name;
    public RealmList<ProjectTask> tasks = new RealmList<>();
}

File: source/examples/Schemas/ProjectAndTask.kt
open class ProjectTask(
    @Required
    var name: String = "",
    var assignee: String? = null,
    var progressMinutes: Int = 0,
    var isComplete: Boolean = false,
    var priority: Int = 0,
    var _partition: String = ""
): RealmObject()

open class Project(
    @Required
    var name: String = "",
    var tasks: RealmList<ProjectTask> = RealmList(),
    var _partition: String = ""
): RealmObject()
